pkg/types/tuf/v0.0.1/tuf_manifest_v0_0_1_schema.json

```
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.sigstore.dev/types/tuf/tuf_manifest_v0_0_1_schema.json",
+    "title": "tuf targets payload v0.0.1 Schema",
+    "description": "Schema for tuf manifests",
+    "type": "object",
+    "properties": {
+        "_type": {
+            "description": "Type identifier",
+            "type": "string",
+            "enum": ["root", "targets", "snapshot", "timestamp"]
+        },
+        "version": {
+            "description": "File version",
+            "type": "integer"
```

```suggestion
            "type": "integer",
            "minimum": 1
```
per https://github.com/theupdateframework/specification/blob/master/tuf-spec.md